src/core/agent/services/identifierService.ts

```diff
@@ -127,21 +126,19 @@ class IdentifierService extends AgentService {
 
   @OnlineOnly
   async createIdentifier(
-    metadata: Omit<
-      IdentifierMetadataRecordProps,
-      "id" | "createdAt" | "signifyName"
-    >
+    metadata: Omit<IdentifierMetadataRecordProps, "id" | "createdAt">
   ): Promise<CreateIdentifierResult> {
     const startTime = Date.now();
 
     if (!identifierTypeThemes.includes(metadata.theme)) {
       throw new Error(`${IdentifierService.THEME_WAS_NOT_VALID}`);
     }
 
-    const signifyName = uuidv4();
-    const operation = await this.props.signifyClient
-      .identifiers()
-      .create(signifyName); //, this.getCreateAidOptions());
+    let name = `${metadata.theme}:${metadata.displayName}`;
+    if (metadata.groupMetadata) {
+      name = `${metadata.theme}:${metadata.groupMetadata.groupId}:${metadata.displayName}`;
+    }
+    const operation = await this.props.signifyClient.identifiers().create(name); //, this.getCreateAidOptions());
```

Comment:
What kind of error do we get if we try to create an identifier with a duplicate name? (so same theme)

Just to make sure the UI can handle it

Reply:
When we try to create an identifier with a duplicate name, the Keria returns an error 400
`400 Bad Request - {"title": "AID with name ${theme}:${displayName} already incepted"} `